### PR TITLE
UI: Change and rename "upstream/downstream" view

### LINKED ISSUE #466: Links: show both A->B and B->A in the Links panel

### PR DESCRIPTION
**Pull request by flomnes**
# Map
image
# Before
image
# After
image

### CODE REVIEW

File: src/ui/simulator/toolbox/input/connection.cpp [hunk 1]
```diff
@@ -216,49 +216,67 @@ void Connections::update()
 
     // UPSTREAM / DOWNSTREEAM
     // Root Node
-    localRootId = pListbox->AppendItem(rootId, wxString(wxT("Upstream / Downstream")), 2);
+    localRootId = pListbox->AppendItem(rootId, wxString(wxT("By area")), 2);
     pListbox->SetItemBold(localRootId, true);
+
+    std::map<Data::AreaName, std::vector<std::pair<Data::AreaLink*, bool>>> areaLinks;
```
Comment (guilpier-code) [inline src/ui/simulator/toolbox/input/connection.cpp]: **areaLinks** could be renamed into **areaToListOfLinks**, as **areaLinks** is a bit vague

**Review by guilpier-code — CHANGES_REQUESTED** [review]

**Comment by flomnes** [comment]
With this PR, the flat view would become obsolete. Remove it ?

File: src/ui/simulator/toolbox/input/connection.cpp (hunk 1 above)
Comment (flomnes) [reply src/ui/simulator/toolbox/input/connection.cpp]: done

**Comment by flomnes** [comment]
@guilpier-code Can we merge this ?

**Comment by JMJ-rte** [comment]
Oui, on peut merger

**Pull request merged**